helm-chart/binderhub/templates/deployment.yaml

```
@@ -102,6 +102,8 @@ spec:
           value: {{ .Values.jupyterhub.hub.services.binder.oauth_client_id | quote}}
         - name: JUPYTERHUB_OAUTH_CALLBACK_URL
           value: {{ .Values.jupyterhub.hub.services.binder.oauth_redirect_uri | quote}}
+        - name: JUPYTERHUB_ALLOW_NAMED_SERVERS
+          value: {{ .Values.jupyterhub.hub.allowNamedServers }}
```

```suggestion
          value: {{ .Values.jupyterhub.hub.allowNamedServers | quote }}
```
Needs to be quoted otherwise `true` is converted to a boolean, but it needs to be kept as a string since it's an env-var.